test/uk/gov/hmrc/teamsandrepositories/controller/DeletedRepositoriesControllerSpec.scala

```diff
@@ -94,17 +95,39 @@ class DeletedRepositoriesControllerSpec
         DeletedGitRepository("Foo", now)
       )
 
-      when(mockDeletedRepositoriesPersistence.find(any(), any()))
+      when(mockDeletedRepositoriesPersistence.find(any(), any(), any(), any()))
         .thenReturn(Future.successful(expectedModel))
 
-      val request: FakeRequest[AnyContentAsEmpty.type] = FakeRequest(GET, getRoute(Some("Foo")))
+      val request: FakeRequest[AnyContentAsEmpty.type] = FakeRequest(GET, getRoute(name = Some("Foo")))
 
       val result = route(app, request).value
 
       status(result) mustBe OK
       contentAsJson(result) mustBe Json.toJson(expectedModel)
 
-      verify(mockDeletedRepositoriesPersistence).find(eqTo(Some("Foo")), eqTo(None))
+      verify(mockDeletedRepositoriesPersistence).find(eqTo(Some("Foo")), eqTo(None), eqTo(None), eqTo(None))
     }
+
+    "get all deleted services" in {
+
+      implicit val reads: OFormat[DeletedGitRepository] = DeletedGitRepository.apiFormat
```

Comment:
these tests should be asserting expected json, to ensure the internal apiFormat is correct

Reply:
fixed